tests/proot-test.nix

```diff
@@ -3,33 +3,9 @@
 { pkgs ? import <nixpkgs> { } }:
```

Comment:
That is still not the same nixpkgs. Is a bit of nitpicking, so I am fine with it, at least it is the same version of termux :)

Reply:
Yeah, I pushed that fix to the wrong branch, should be `(import ../pkgs/lib/load-nixpkgs.nix {})`.